### ISSUE TITLE
When resizing the log viewer it preserves the top rather than the bottom of the visible area

### ISSUE THREAD

**Issue by nriley** [talonvoice/talon#594]
Since you're most often looking at the bottom of the log it would make more sense to resize preserving the bottom of the visible area.

**Comment by lunixbochs** [comment]
yeah I noticed this one, it's an egui behavior that I haven't figured out how to override yet
